docs/rst/notes/forthcoming_version.rst

```diff
@@ -9,8 +9,16 @@ Forthcoming Version
 The next release will include the following **Features**:
 
 * :ref:`Forwarding Routes <user_manual_configuration_forwarding_routes>`.
-* :ref:`Remove Unused Entities <user_manual_configuration_remove_unused_entities>`
+* :ref:`Remove Unused Entities <user_manual_configuration_remove_unused_entities>`.
+* :ref:`Manual Topics <user_manual_configuration_manual_topics>`.
+* :ref:`Max Transmission Rate <user_manual_configuration_max_tx_rate>`.
+* :ref:`Max Reception Rate <user_manual_configuration_max_rx_rate>`.
+* :ref:`Downsampling <user_manual_configuration_downsampling>`.
 
 The next release will include the following **Bugfixes**:
 
 * Save the instance handle data for keyed topics.
+
+The next release will include the following **Internal Adjustments**:
```

Comment:
NIT: I wouldn't say this is an internal adjustment, it directly affects the user.